Public/Sdk/Experimental/NodeJs/Yarn/yarn.dsc

```diff
@@ -114,35 +114,16 @@ export function install(args: Arguments) : Result {
 
             const qAuthMaterial = {
                 environmentVariables: [{name: "QAUTHMATERIALROOT", value: qAuthMaterialRoot.path}],

```

Comment:
Should we make this environment variable a passthrough variable as well then?

Reply:
yes, I think so, and it has already been marked as passthrough by GBR. I will remove this line.